pyp2rpm/filters.py

```diff
@@ -71,11 +74,49 @@ def macroed_url(url):
     return url
 
 
+def rpm_version_410(version, use_macro=True):
+    if use_macro:
+        default = '%{pypi_version}'
+    else:
+        default = version
+    re_match = re.compile(
+        r"(\d+\.?\d*\.?\d*\.?\d*)\.?((?:a|b|rc|dev)\d*)").search(
+            version)
+    if not re_match:
+        return default
+    rpm_version, rpm_suffix = re_match.groups()
+    if rpm_suffix.startswith("dev"):
+        return '{}~~{}'.format(rpm_version, rpm_suffix)
```

Comment:
Do you think we actually *should* support -dev versions? Doing so makes this non-standard double-tilde necessary, and it's at least not covered by the Fedora packaging guidelines at all.

Reply:
The double tilde is what the Python dist RPM dependency generator does for dev.